### ISSUE TITLE
Accessibility updates

### ISSUE THREAD

**Issue by ggeisler** [projectblacklight/spotlight#2348]
As part of a current workcycle, the Stanford team intends to identify and address accessibility issues. This will be a high-level story ticket, with more focused and actionable accessibility tickets to follow.

**Comment by ggeisler** [comment]
Closing this because we've made recent accessibility updates using the https://github.com/sul-dlss/exhibits repo to keep track of relevant issues.
